### ISSUE TITLE
Internal voids ignored

### ISSUE THREAD

**Issue by T3P3** [alexrj/Slic3r#1472]
@alexrj Alessandro sorry for the first communication after the TCT show is to re-open an old issue (#216) but I have discovered that handling of internal voids is still a problem (at least in the limited area I have tested so far).

Slicing the new Mendel90 parts, specifically the X-Motor bracket and X-Idler bracket (eg https://github.com/nophead/Mendel90/blob/master/dibond/stls/x_idler_bracket.stl) the nut trap at the top of the model is ignored and treated as a solid part. The link to the part allows you to use a wireframe to see the internal void for the nut trap, also the attached picture which shows a visualisation of the Gcode output by slic3r 0.9.10b:

![slic3r_internal_void_filled](https://f.cloud.github.com/assets/1642947/1261529/50433e52-2c2f-11e3-9a78-1ae7803f6a1a.PNG)

And a visualisation of the code as produced by an alternative slicing program:
![alternative_correct_handling_of_internal_void](https://f.cloud.github.com/assets/1642947/1261528/5039af36-2c2f-11e3-8478-8e4a8169b00d.PNG)

In summary my slicing settings:
2+ perimeters
2/3 bottom/top layers
0.2 line infil
no support.
but I have tried some variations on these without seeing a difference.

Cheers Tony
